gradle/versions.gradle

```diff
@@ -190,6 +190,7 @@ dependencyManagement {
     dependency 'org.web3j:besu:4.8.9'
     dependency 'org.web3j:core:4.8.9'
     dependency 'org.web3j:crypto:4.8.9'
+    dependency 'org.web3j:eea:4.9.3'
     dependency 'org.web3j:quorum:4.8.4'
 
```

Comment:
Ideally we should aim to keep these consistent as possible. 

```
dependency 'org.web3j:besu:4.9.3'
dependency 'org.web3j:core:4.9.3'
dependency 'org.web3j:crypto:4.9.3'
dependency 'org.web3j:eea:4.9.3'
dependency 'org.web3j:quorum:4.9.0'
```

Reply:
`org.web3j:quorum:4.9.3` does not exist, which is why 4.9.0 is used.

Reply:
I agree completely with you guys, however there is an issue in web3j for versions >= 4.9.0 (https://github.com/web3j/web3j/issues/1747). I have put a comment in the PR description about why 4.9.3 is just for that package.

Happy to upgrade to a potential 4.9.4 (and 4.9.0 for `quorum` ;) ) once the bugfix for https://github.com/web3j/web3j/issues/1747 is implemented. Not sure if it will happen soon enough though.